app/assets/stylesheets/base/globals.scss

```diff
@@ -2,8 +2,9 @@ body {
   font-family: "Source sans Pro", sans-serif;
   font-weight: 400;
   overflow-x: hidden;
+  background-color: #eee;
```

Comment:
Color literals like `#eee` should only be used in variable declarations; they should be referred to via variable everywhere else.